app/src/kotlin/java/co/omise/android/example/CheckoutActivity.kt

```diff
@@ -41,7 +43,7 @@ class CheckoutActivity : AppCompatActivity() {
     companion object {
 
         private const val TAG = "CheckoutActivity"
-        private const val PUBLIC_KEY = "[PUBLIC_KEY]"
+        private const val PUBLIC_KEY = "pkey_5xjdtti64j5cwzxa9xo"
```

Comment:
Is it required to include a valid `pkey`?

Reply:
oh yes sorry i should remove